server.js

```diff
@@ -1,5 +1,6 @@
 //Imports
 var express = require('express');
+var bodyParser = require('body-parser');
```

Comment:
Why do we need the bodyParser in this example?

Reply:
Actually we don't , I just noticed that. There was a piece of code that was using it as middleware but no request body is actually used.